src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala

```diff
@@ -151,6 +153,7 @@ class DepositServlet(app: EasyDepositApiApp)
         uuid <- getUUID
         path <- getPath
         _ <- isMultipart
+        _ <- canUpdate(uuid)
```

Comment:
The `canDelete` (isDeletable) is called by the `DepositDir` class. This servlet class should only unwrap the http-request and wrap up the results in a response, it should not contain execution logic. Must admit I violated this rule above by calling `app.checkDoi` which should be moved into `writeDataMetadataToDeposit`. In this particular case I still have my doubts I did get this right.

A side note probably a bit beyond the scope of this issue: I made a bit of a mess in the past from reading de deposit properties and getting the status out of it. The problem was avoiding reading again and again and not getting into trouble with changes.